### PR TITLE
Fixed PHPLIB-108: Library relies on BSON being a vendor prefix, while our vendor should be MongoDB

### PR DESCRIPTION
**Pull request by derickr**
https://jira.mongodb.org/browse/PHPLIB-108

### CODE REVIEW

**Comment by jmikola** [comment]
This needs to wait until [PHPC-309](https://jira.mongodb.org/browse/PHPC-309) is implemented and released, since we'll also need to bump the ext-mongo dependency.

File: src/Collection.php [hunk 1]
```diff
@@ -856,6 +869,17 @@ public function updateOne(array $filter, array $update, array $options = array()
     }
 
     /**
+     * Internal helper for running a command
+     * @internal
+     */
+    final protected function _runCommand($dbname, array $cmd, ReadPreference $rp = null)
```
Comment (jmikola) [inline src/Collection.php]: Why was this method added?

Comment (derickr) [inline src/Collection.php]: That wasn't me... NFC where it came from.

**Comment by jmikola** [comment]
Rebased and merged in #22.

**Pull request closed without merging**